Tensile/Source/client/source/ResultFileReporter.cpp

```diff
@@ -35,19 +35,26 @@ namespace Tensile
         std::shared_ptr<ResultFileReporter>
             ResultFileReporter::Default(po::variables_map const& args)
         {
-            return std::make_shared<ResultFileReporter>(args["results-file"].as<std::string>(),
-                                                        args["csv-export-extra-cols"].as<bool>(),
-                                                        args["csv-merge-same-problems"].as<bool>());
+            return std::make_shared<ResultFileReporter>(
+                args["results-file"].as<std::string>(),
+                args["csv-export-extra-cols"].as<bool>(),
+                args["csv-merge-same-problems"].as<bool>(),
+                args["performance-metric"].as<PerformanceMetric>());
         }
 
         ResultFileReporter::ResultFileReporter(std::string const& filename,
                                                bool               exportExtraCols,
-                                               bool               mergeSameProblems)
+                                               bool               mergeSameProblems,
+                                               PerformanceMetric  performanceMetric)
             : m_output(filename)
             , m_extraCol(exportExtraCols)
             , m_mergeSameProblems(mergeSameProblems)
+            , m_performanceMetric(performanceMetric)
         {
-            m_output.setHeaderForKey(ResultKey::ProblemIndex, "GFlops");
+            if(m_performanceMetric == PerformanceMetric::CUEfficiency)
+                m_output.setHeaderForKey(ResultKey::ProblemIndex, "GFlopsPerCU");
+            else // (m_performanceMetric == PerformanceMetric::Overall)
```

Comment:
Just checking if this is still valid for 'Best' or 'Count' enum values

Reply:
Only CUEfficiency and Overall make sense at the benchmarking stage. I updated the comment (and added one in LogReporter.hpp) clarifying this.

The other option is to add an error/warning message if something other than one of these two values is passed in.